packages/editor/src/lib/editor/shapes/ShapeUtil.ts

```diff
@@ -89,13 +89,6 @@ export abstract class ShapeUtil<Shape extends TLUnknownShape = TLUnknownShape> {
 	 */
 	canScroll: TLShapeUtilFlag<Shape> = () => false
 
-	/**
-	 * Whether the shape should unmount when not visible in the editor. Consider keeping this to false if the shape's `component` has local state.
-	 *
-	 * @public
-	 */
-	canUnmount: TLShapeUtilFlag<Shape> = () => true
```

Comment:
We only used this to determine whether we should cull shapes or not. The old logic unmounted the culled shapes, which we didn't want for some embeds since they had local state. Since we just do `display: none` with the new logic we can probably remove `canUnmount` completely.